src/Donations/Micros/PaymentStatus/ThankyouMessage.tsx

```diff
@@ -49,17 +48,16 @@ function ThankyouMessage({
       : null;
 
   // EXAMPLE: Your 50 trees will be planted by AMU EcoVillage Project, Ethiopia in Ethiopia.
-  const donationProjectMessage = donation.destination
-    ? " " +
-      t("common:yourTreesPlantedByOnLocation", {
-        treeCount: getFormattedNumber(
-          i18n.language,
-          Number(donation.treeCount)
-        ),
-        projectName: donation.destination.name,
-        location: t("country:" + donation.destination.country.toLowerCase()),
-      })
-    : null;
+  const donationProjectMessage =
+    donation.destination && donation.units
+      ? " " +
+        t("common:restorationDonationUsage", {
+          units: donation.units,
```

Comment:
Don't we want to format `donation.units` any more at all in the UI (I know you do not know if this is a number of trees, square meter or amount of currency)? In `src/Donations/LeftPanel/TransactionSummary.tsx` we also use a function `getAdditionalInfo` to format the number depending on the unitType.